### ISSUE TITLE
Implement gradient clipping

### ISSUE THREAD

**Issue by johancc** [johancc/Styler#8]
There seems to be exploding gradients when training on large data sets or many epochs
